deepchem/feat/molecule_featurizers/__init__.py

```diff
@@ -14,3 +15,4 @@
 from deepchem.feat.molecule_featurizers.smiles_to_image import SmilesToImage
 from deepchem.feat.molecule_featurizers.smiles_to_seq import SmilesToSeq, create_char_to_idx
 from deepchem.feat.molecule_featurizers.mol_graph_conv_featurizer import MolGraphConvFeaturizer
+from deepchem.feat.molecule_featurizers.molgan_featurizer import MolGanFeaturizer
```

Comment:
We should also add a link to `MolGanFeaturizer` in the docs. Can you add a listing to docs/source/api_reference/featurizers.rst to add autogenerated docs? This should hopefully be clear how to add by modifying the autodoc commands for the other docs.

The nice step of adding this is we'll be able to read the `MolGanFeaturizer` docs on the docs website that way :)

Reply:
will do

Reply:
Anybody knows good rst extension for VS Code? I have installed reStructuredText and just highlighted the whole file red.